monero-GUI-guide.md

```diff
@@ -199,24 +199,23 @@ and privacy needs.\
 (1) **Simple mode:** In this mode your wallet will display a simple
     interface with easy access to basic functionality like sending,
     receiving, transactions history, address book, accounts, and
-    merchant mode, as well as hardware wallet support. It will connect
-    to a third-party server (a remote node), and it will not start a
-    local node in your computer (the blockchain will not be downloaded).
-    Choose this option if you don't have enough space to store the
-    blockchain file (at least 90 GB), if you don't require maximum
-    privacy and you accept the privacy risks of using a remote node to
-    connect to the Monero network.\
+    merchant mode, as well as hardware wallet support. It will start a
+    local node on your computer, but the blockchain will not be downloaded
+    Instead, the local node will forward requests to a user-run remote
+    node known as a bootstrap node. Choose this option if you don't have
```

Comment:
I'm not sure abut using 'bootrstap node' here. We usually call "bootstrap mode" when a user is synchronizing a local node (downlading a blockchain), but uses a remote node to be able to transact immediately. I think using "bootstrap" in this context would create confusion. Actually, i would avoid the word bootstrap at all, which is usually confusing for both users and translators.

Reply:
It was already being called a "bootstrap node" in other places in the document so the changes I suggested were 1) to make the language more consistent and 2) to avoid calling the nodes "third party servers" which imo isn't quite right for a peer-to-peer network. Even though the wallet and daemon do have a client / server relationship, I think it's best to stick to node / wallet language. Just my 2c.

I don't personally have a strong opinion on using "bootstrap node" vs "remote node". If you think 'bootstrap' is confusing to users, we should change it, but I think it should be done in separate PR so we can remove the phrase "bootstrap node" from the entire document at once.

As far as avoiding the word entirely, as you know the wallet itself refers to this mode as "Simple mode (bootstrap)" so I don't see how that's feasible. We can definitely drop "bootstrap node" though.

Reply:
Well, now I'm looking more closely and the GUI also uses the phrase "bootstrap node". See [here](https://raw.githubusercontent.com/monero-ecosystem/monero-GUI-guide/master/media/wizard_5-daemon-settings.png).

Reply:
My point is that basically calling a remote node "bootstrap node" it's confusing, because we a mode called "bootstrap mode" which does something completely different (as we see in the next paragraph).

> If you think 'bootstrap' is confusing to users, we should change it,

We already started to use the "bootstrap" word much less, because it's very technical for users and translators, which have a lot of troubles with that particular word. I've changed the naming in the downloads page of getmonero and other places, but i agree we should do more. Problem is that it's hard to coordinate between GUI devs, CLI devs and others. 

> We can definitely drop "bootstrap node" though.

Yes, that's my point. We should definitely use "remote node" instead. I would really like to get rid of the word "bootstrap" in general, but that's a discussion for another time :)

> Well, now I'm looking more closely and the GUI also uses the phrase "bootstrap node". See here.

True, but that's strictly related to the "bootstrap mode" feature.

Reply:
Made some changes to the wording based on your feedback. Included changes to the advanced mode description, to keep it consistent.

If you prefer to stick with "remote node (bootstrap node)" we can. I think the parentheses implies they are the same whereas "bootstrap remote node" implies a subtype of remote node. Either seems acceptable to me but I somewhat favor the latter.

Whatever you decide, I'll go over the full the document to check for consistency.

Reply:
Looks good. Thanks!